ssl-config/src/main/java/com/palantir/remoting/ssl/SslConfiguration.java

```diff
@@ -30,13 +30,19 @@
 @Value.Style(visibility = ImplementationVisibility.PACKAGE)
 public abstract class SslConfiguration {
 
-    private static final String DEFAULT_STORE_TYPE = "JKS";
+    public enum StoreType {
```

Comment:
Typing as enum seems better/more elegant than using String.

Back-compat break for Java code, but not for JSON/serialization (since "JKS" and "PKCS12" enum constants have the same String representation as constants that were used before, objects are still serialization-compatible)